test/cmdlineTests/abiencoderv2_no_warning/input.sol

```diff
@@ -4,6 +4,8 @@ pragma abicoder v2;
 
 contract C {
     struct S { uint x; }
+    event ev(uint y);
+    error err(uint z, uint w);
     function f(S memory) public pure {
     }
```

Comment:
@hrkrshnn 
> Could you add a test similar to https://github.com/ethereum/solidity/tree/develop/test/cmdlineTests/abiencoderv2_no_warning or simply modify the contract to include an event and error?

I think we should really have a dedicated test for `--hashes`.

Reply:
Maybe this test is not named properly, but the `args` file only has `--hashes`!

Reply:
True! It's still meant to test something different though: 0973ae751af1c6a010fafe193c5e099c09eb8e7a. I think it could have been done using another other option and it might change in the future due.

We could use a name that implies that it tests both, but at this point, I'd rather have two tests even if they're very similar.